cypress/integration/edit-post.test.ts

```diff
@@ -0,0 +1,126 @@
+import { HttpMethod } from "@lib/http-method";
+import { StatusCodes } from "http-status-codes";
+import { TEST_POST_2_UUID } from "../support/constants";
+
+describe("Edit post", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should allow a user edit a post from the account page", () => {
+    cy.login();
+    cy.visit("/account");
+
+    cy.intercept(HttpMethod.PUT, "/api/post/*").as("editPost");
+    cy.intercept(HttpMethod.GET, "/api/user/*").as("getUserPosts");
+
+    // Check if editing the post works
+    cy.findAllByRole("button", { name: /edit post/i })
+      .eq(0)
+      .click();
+
+    cy.findByRole("dialog")
+      .get('[id="description"]')
```

Comment:
Best practice would be to do `cy.findByRole("textbox", { name: /description/i })` or `cy.findByLabelText(/description/i)` to select the input using its field label (to ensure best accessibility practices are used). Kind of a nit pick though.

(also see the one for price)